wherehows-common/src/main/java/wherehows/common/utils/JobsUtil.java

```diff
@@ -96,6 +111,23 @@ private static String jobNameFromFile(File file) {
   }
 
   /**
+   * Returns a map of job name to job properties which are enabled.
+   */
+  public static Map<String, Properties> getEnabledJobs(String dir) {
+    Map<String, Properties> jobs = new HashMap<>();
+    for (File file : new File(dir).listFiles()) {
+      if (file.getAbsolutePath().endsWith(".job")) {
+        Properties prop = getResolvedProperties(file.toPath());
+        if (!prop.containsKey(Constant.JOB_DISABLED_KEY)) {
+          // job name = file name without the extension.
+          jobs.put(jobNameFromFile(file), prop);
+        }
+      }
+    }
+    return jobs;
+  }
+
+  /**
    * Returns a map of job name to job properties which are enabled and of certain type.
    */
   public static Map<String, Properties> getEnabledJobsByType(String dir, String type) {
```

Comment:
Do we still need this function?

Reply:
There is one other usage in TreeBuilderActor:    
treeJobList = JobsUtil.getEnabledJobsByType(ETL_JOBS_DIR, TREE_JOB_TYPE);